.github/workflows/CI.yml

```diff
@@ -0,0 +1,95 @@
+on:
+  push:
+    branches: [master]
+  pull_request:
+
+name: Continuous integration
+
+jobs:
+  check:
+    name: Check
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rust:
+          - stable
+          - nightly
```

Comment:
I don't particularly care about nightly, as I am trying to use "mainline features only" for now (kind of like avoiding python 3.10/4 funcs until it stabilises/my knowledge expands).

Would it be meaningful then to remove nightly from here? Don't think so, but heh.

Reply:
yep, fair enough. if you're targeting stable, no point testing the nightly toolchain.